src/js/control-bar/seek-to-live.js

```diff
@@ -39,19 +39,19 @@ class SeekToLive extends Button {
       className: 'vjs-seek-to-live-control vjs-control'
     });
 
-    this.contentEl_ = Dom.createEl('div', {
-      className: 'vjs-seek-to-live',
-      innerHTML: `<span class="vjs-control-text">${this.localize('Stream Type')}\u00a0</span>${this.localize('LIVE')}`
+    this.textEl_ = Dom.createEl('span', {
+      className: 'vjs-seek-to-live-text',
+      innerHTML: this.localize('LIVE')
     }, {
-      'aria-live': 'off'
+      'aria-hidden': 'true'
```

Comment:
When does this button get `aria-hidden`? It should not be displayed when it's not relevant (and "display: none" prevents it from being visible to a screen reader as well as to a sighted user), and if it is disabled it is better to mark it as disabled (or `aria-disabled`) than `aria-hidden`.

Reply:
I think the idea is that the span that says "LIVE" is only for sighted users and SR/keyboard users get the same information elsewhere (like the control text)

Reply:
exactly as @gkatsev said, that is the idea